scripts/lib-franklin.js

```diff
@@ -366,13 +366,43 @@ export function decorateSupScript(string, result = []) {
   return result;
 }
 
+export function getInfo() {
+  const [, country, language] = window.location.pathname.split('/');
+  return {
+    country,
+    language,
+    productDB: new URL('/products.json', window.location).pathname,
```

Comment:
i would move the URL constructions into a small local-scope helper function for readability and easier adjustment:

```
  const getUrlPath = (path) => new URL(path, origin).pathname;
  
  return {
    country,
    language,
    productDB: getUrlPath('/products.json'),
    productSupport: getUrlPath(`/${country}/${language}/product-support`),
    queryIndex: getUrlPath(`/${country}/${language}/query-index.json`),
  };
```